### ISSUE TITLE
[Gallery][Skia.Gtk] CheckBox(Material)-Checkmark in Checkbox is not aligned correctly.

### ISSUE THREAD

**Issue by sakshi173** [unoplatform/uno#15368]
### Current behavior

![checkbox issue](https://github.com/unoplatform/uno/assets/74563602/eb836857-85d1-41b3-891f-4063f8904b82)

### Expected behavior

![Checkbox correct](https://github.com/unoplatform/uno/assets/74563602/44ff6ff1-5b43-4aae-ba18-034fd588b364)

### How to reproduce it (as minimally and precisely as possible)

Open Uno Gallery APP
Tap Burger Menu on the left top right.
Tap "CheckBox" from the "Ui Component" list.
Go to Material section.
Notice that Checkmark in Checkbox is not aligned correctly.

### Works on UWP/WinUI

None

### Affected platforms

Skia (GTK on Linux/macOS/Windows)

**Comment by jeromelaban** [comment]
@Youssef1313 I recall we had an issue about this, do you remember which one?

**Comment by Youssef1313** [comment]
@jeromelaban Yes. Originally, before doing layouting fixes, the glyph wasn't positioned correctly and this was worked-around in Uno.Themes.

After the layouting in Uno, the workaround caused it to not be positioned correctly and the workaround was removed. Issue: https://github.com/unoplatform/Uno.Themes/issues/1197

Not sure why this is back now.

**Comment by Youssef1313** [comment]
@sakshi173 Can you please confirm if this is still an issue?

**Comment by sakshi173** [comment]
@Youssef1313 , I am not able to reproduce this issue now, due to that closing it.
